CHANGELOG.md

```diff
@@ -22,13 +22,18 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - `CustomDiscreteParameter` does not allow duplicated rows in `data` anymore
 - De-/activating Polars via `BAYBE_DEACTIVATE_POLARS` now requires passing values
   compatible with `strtobool`
+- `allow_repeated_recommendations` has been renamed to 
+  `allow_recommending_already_recommended` (now available for `Campaign` instead of
```

Comment:
The parenthesis was thought only as an aid for people to find it. The actual move of the flag (and all other flags) is already under `deprecations`. So if I only remove the parenthesis, it would be a bit weird to have the info about the move of that flag mentioned again, but not for the others. So let me remove it entirely and only mention the renaming.